AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4570] Update variant options so that they work on github.io

## Ticket(s)

**Jira Ticket**
- https://issues.ama-assn.org/browse/EWL-4570

## Description
Update Variant Patterns MD files for baseurl flexibility.

Should I try to FIX the Ribbon Drowndown variants links? YOU TELL ME. :)


## To Test
Open pattern -- click on the top right gear and click on the `Show Pattern Info` and view. Verify all links go to their respectable destination.

- [ ]  http://localhost:3000/?p=molecules-info-card
- [ ]  http://localhost:3000/?p=molecules-partner-info
- [ ]  http://localhost:3000/?p=molecules-promo (ALL VARIANTS)
- [ ]  http://localhost:3000/?p=organisms-masthead
- [ ]  http://localhost:3000/?p=organisms-membership-with-cta
- [ ]  http://localhost:3000/?p=organisms-membership

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
- [ ]  https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-info-card
- [ ]  https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-partner-info
- [ ]  https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-promo (ALL VARIANTS)
- [ ]  https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-masthead
- [ ]  https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-membership-with-cta
- [ ]   #https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-membership


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
